scripts/newsexsystem.gd

```diff
@@ -70,6 +73,8 @@ class member:
 	var vagTorn = false
 	var assTorn = false
 	var actionshad = {addtraits = [], removetraits = [], samesex = 0, samesexorgasms = 0, oppositesex = 0, oppositesexorgasms = 0, punishments = 0, group = 0, incest = 0, incestorgasms = 0}
+	var succubusdrain = 0 #ralphC
```

Comment:
`succubusdrain` fulfills the roll of a boolean but uses the calculations of an integer. Either way it is unnecessary to store this value in the class as `succubus_spunked()` can easily absorb the loops checking for "Succubus" prior to the call by passing the container of participants as a function argument.

Reply:
I have no doubt you are correct, but I'm leaving it as is for expediency.  I'll aim to streamline as noted when I come back to this script to add a new similar mechanic.